twilight-gateway/src/config.rs

```diff
@@ -87,18 +84,11 @@ impl Config {
     ///
     /// Panics if loading TLS certificates fails.
     pub fn new(token: String, intents: Intents) -> Self {
-        Self::builder(token, intents).build()
-    }
-
-    /// Create a builder to customize a shard's configuration.
-    ///
-    /// # Panics
-    ///
-    /// Panics if loading TLS certificates fails.
-    pub fn builder(token: String, intents: Intents) -> ConfigBuilder {
```

Comment:
Why is this removed?

Reply:
The type inferrence doesn't play nicely with this method, you'd have to write `Config::builder::<QueueImplementor>(token, intents)`

Reply:
> The type inferrence doesn't play nicely with this method, you'd have to write `Config::builder::<QueueImplementor>(token, intents)`

Ugh, let's just remove it then. It's not like this has any advantage over using `ConfigBuilder` directly anyways.